contrib/initramfs/scripts/zfs.in

```diff
@@ -436,7 +436,11 @@ decrypt_fs()
 
 			# Prompt with ZFS tty, otherwise
 			else
+				# Setting "printk" temporarily to "7" will allow prompt even if kernel option "quiet"
+				storeprintk="$(cat /proc/sys/kernel/printk | awk '{print $1}')"
```

Comment:
The `cat` is unnecessary if you just use: `awk '{print $1}' /proc/sys/kernel/printk`

Reply:
Thanks